IFComp/lib/IFComp/Schema/Result/Entry.pm

```diff
@@ -1186,8 +1186,10 @@ sub _create_quixe_page {
     # Create a JS file for this game.
     my $js_file = Path::Class::File->new( $i7_file . '.js' );
     $self->_make_js_file( $i7_file, $js_file );
+    my $toplevel_directory = $self->content_directory . '/';
 
-    my $js_filename = $js_file->basename;
+    my $js_filename = $js_file;
+    $js_filename =~ s/^$toplevel_directory//;
```

Comment:
This would be better written as `my $js_filename = $js_file->relative( $self->content_directory );`

It's a bit cleaner, and also more portable.

Reply:
a-ha, yes thank you